sktime/forecasting/model_selection/tests/test_tune.py

```diff
@@ -131,3 +132,29 @@ def test_rscv(forecaster, param_grid, cv, scoring, error_score, n_iter, random_s
         ParameterSampler(param_grid, n_iter, random_state=random_state)
     )
     _check_cv(forecaster, rscv, cv, param_distributions, y, X, scoring)
+
+
+def test_refit():
+    """Test refit param."""
```

Comment:
Could be nice to add another mini test within this to show that `x_train` and `y_train` are the sizes you expect them to be. (and also that cutoff behaves as expected).